src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -58,43 +120,104 @@ export default (accessToken, userID, opts) => {
         entityIDKeyPath: "_id",
       }
     ),
-    followedFairsLoader: gravityLoader("/me/follow/profiles", {}, { headers: true }),
-    followedPartnersLoader: gravityLoader("/me/follow/profiles", {}, { headers: true }),
+    followedFairsLoader: gravityLoader(
+      "/me/follow/profiles",
+      {},
+      { headers: true }
+    ),
+    followedPartnersLoader: gravityLoader(
+      "/me/follow/profiles",
+      {},
+      { headers: true }
+    ),
     homepageModulesLoader: gravityLoader("me/modules"),
-    homepageSuggestedArtworksLoader: gravityLoader("me/suggested/artworks/homepage"),
-    inquiryRequestsLoader: gravityLoader("me/inquiry_requests", {}, { headers: true }),
+    homepageSuggestedArtworksLoader: gravityLoader(
+      "me/suggested/artworks/homepage"
+    ),
+    inquiryRequestsLoader: gravityLoader(
+      "me/inquiry_requests",
+      {},
+      { headers: true }
+    ),
     lotStandingLoader: gravityLoader("me/lot_standings"),
-    meBidderPositionLoader: gravityLoader(id => `me/bidder_position/${id}/`, {}, { headers: true }),
```

Comment:
Someone's Prettier is busted....

Reply:
Yea, these changes confuse me a little. This file isn't updated very often, and the last few updates to it from early this year didn't introduce any big formatting changes.

However, this diff now, _does_ look correct to me. ie- line length is respected, trailing commas have been added in several places, etc.

Weird? I'm a tad bit confused with pretty sometimes. This is correct now, no?

Reply:
Yup, this is correct. 

LOL, I think this is my fault actually, via [.prettierignore](https://github.com/artsy/metaphysics/commit/7ab2f7293be1bc549581bb658d4c35de474e2ba7#diff-23949df756e1e1316d52098dcf8383f0) -- i wonder why I added this / why your setup didn't respect that config? 

/shrug :)

Reply:
Ha! Keeping each loader to one line might scan better, but idk. Strange! (and good find)

Reply:
Kill it if you feel like it -- seems inconsistent now.

Reply:
Ah ok, the files got moved (an `/api` directory was added), so the `.prettierignore` path isn't correct anymore.